Hernan-Alonso/citas
# [FEAT] - Administrador de citas con Formato convencional

### **What was done:**
- Appointment app, where you can add appointments to your list for the vet.

### **Data needed to test:**
- ?.

### **How to test:**
1. Complete the form with the correct values.
2. Submit the form
3. See the card at the right column with the data.

1. Submit the form without completing all data.
2. Error message should appear
